src/commands/deploy.ts

```diff
@@ -0,0 +1,21 @@
+import { Command } from '@oclif/command'
+
+import generateDeploy = require('../lib/commands/generate-deploy')
+
+import { GenerateDeployFlags } from './generate'
+
+export default class Deploy extends Command {
+  static description =
+    'Generate Deploy Spec from environment configuration and deploy'
+
+  static flags = GenerateDeployFlags
+
+  static args = []
+
+  async run() {
+    const { flags } = this.parse(Deploy)
```

Comment:
We use semi-colons at end of lines: https://github.com/balena-io/process/blob/master/process/TypeScript_Coding_Guide.md#semicolons